src/org/labkey/test/tests/AdminConsoleTest.java

```diff
@@ -157,10 +160,11 @@ public void testAppAdminRole()
         assertElementPresent("expect to return to admin console", siteAdminLoc, 1);
 
         //authentication
-        goToAdminConsole().clickAuthentication();
-        assertElementNotPresent("expect 'enable' links to be disabled for appAdmin", Locator.linkWithText("enable"));
-        assertElementNotPresent("expect 'configure' links to be disabled for appAdmin", Locator.linkWithText("configure"));
-        clickButton("Done");
+        LoginConfigurePage configurePage = goToAdminConsole().clickAuthentication();
+        List<LoginConfigRow> configRows = configurePage.getPrimaryConfigurations();
+        assertFalse("expect 'edit' links not to be available for auth configs", configRows.stream().anyMatch(a-> a.canEdit()));
+        assertFalse("expect 'add configuration' menu to be absent for AppAdmin", configurePage.canAddConfiguration());
+        configurePage.clickSaveAndFinish();
```

Comment:
Wait until the product fix is in. This button might become disabled for app admins or something else that would require a followup test fix.